python/fusion_engine_client/messages/device.py

```diff
@@ -0,0 +1,47 @@
+import struct
+
+from construct import (Struct, Int64ul, Int16ul, Int8ul, Int16sl, Padding, this, Bytes, PaddedString)
+
+from ..utils.construct_utils import AutoEnum, construct_message_to_string
+from ..utils.enum_utils import IntEnum
+from .defs import *
+
+class SystemStatusMessage(MessagePayload):
+    """!
+    @brief System status elements.
+    """
+    MESSAGE_TYPE = MessageType.SYSTEM_STATUS
+    MESSAGE_VERSION = 0
+
+    INVALID_TEMPERATURE = -32768
+
+    SystemStatusMessageConstruct = Struct(
```

Comment:
Since these construct members are nested within the message classes, I actually don't particularly like the pattern of naming them `FooConstruct` vs simply a generic `Construct`, `SerializationConstruct`, etc. If we use a single name, then it becomes a little easier to maintain the pack/unpack/str code between all the classes since they'll have that in common.

It doesn't come up often, but it has been a bit tedious a few times when we've had to change a pattern across all messages, and we have to change this name for each one.

@axlan thoughts?

Reply:
Since the messages have grown somewhat organically, it probably wouldn't hurt to take a step back at some point and try to rethink the inheritance design. In that sense it probably makes sense to refactor this, but it's outside the scope of this PR.